File: ex18.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float peso, altura, imcm,imcf, m, f;
    char sexo;
    printf("digite o seu peso:");
    scanf("%f", &peso);

    printf("digite o valor da altura:");
    scanf("%f", &altura);

    printf("digite o sexo da pessoa:");
    scanf("%s", &sexo);

   { imcm = peso/ (altura*altura)+m;
    
    if (imcm<20){
     printf("abaixo do peso:");
     
    }
    else if(imcm<25){
    printf("peso ideal") ;

    }}
    //else{
   // printf("acima do peso") ;    
  //  }}
    

  {  imcf = peso/ (altura*altura)+f;
    
    if (imcf<19){
     printf("abaixo do peso:");
     
    }
    else if(imcf<24){
    printf("peso ideal") ;
    }
    else{
    printf("acima do peso") ;    
    }}
    


    return 0;
}

File: python/ex18.py
peso = float(input("entre com seu valor de peso:"))
altura= float(input("entre com sua altura:"))
m=str
f=str
sexo=str(input("diga qual é o seu sexo:"))

imc = peso/ (altura*altura) 

if (m and (imc<20)) or (f and (imc<19)):
   print("abaixo do peso")

elif (m and (20<imc<25)) or (f and (19<imc<24)):
    print("peso ideal")
else:
    print("acima do peso")

 

